### ISSUE TITLE
Fixed .epub File for Interactive Kiosk Use

### ISSUE THREAD

**Issue by bshary** [edrlab/thorium-reader#1604]
We are using the Thorium Reader product on an Ideum Drafting II touchscreen display for an interactive, FIXED .epub file. The file uses buttons to navigate the content, and has simple animations and video content.  
 
We had some difficulty with the file itself and full screen viewing. When using the provided fixed scale options in the Thorium Reader, the file either flows outside of the screen view, or gives us a white border around the material. We'd like the content to go edge to edge to ensure optimal viewing experience on the kiosk. Is there any way to use the Thorium product and adjust it format correctly for "full screen" mode?

We've tried a variety of other .epub reader programs and other readers either re-formatted the pages incorrectly, disabled the button navigation, or disabled the animated content. Thorium is the closest (and most user friendly) we've found, but we're hoping to refine the scaling issues. I am not a programmer, so would love to work with the Github folks on any improvements that may make Thorium ideal for this use case. 

The kiosk screen we're using is a 43" diagonal touch screen display operating on Windows 10 Pro, 4K UHD (3840x2160) LCD display.

Thank you in advance for any help!

**Comment by danielweck** [comment]
> the file either flows outside of the screen view

This sounds like a bug.
Are you experiencing this behaviour consistently?
Is this with the latest official release v1.7.3?
Have you tried the latest 1.7.4 alpha build?
https://github.com/edrlab/thorium-reader/releases

> or gives us a white border around the material. We'd like the content to go edge to edge

This is a feature request for the existing "full screen" mode. The UX problem we need to solve here is that once the reader window enters such "proper" fullscreen mode, typically users would need to be able to switch back to the regular view, and of course normal Thorium usage would typically involve closing the reader window to re-enter the library window (bookshelf). Am I right to assume that your kiosk touchscreen-exclusive use case consists in a locked-in display of a specific publication (in other words, regular users who interact via touch cannot navigate into other parts of the Thorium GUI, let alone Windows OS UI). In that case, I would assume that an administrator would be able to interact with a keyboard instead of touch, affording them access to app-level or system-level GUI?

**Comment by bshary** [comment]
Yes, the publication is intended to be a locked-in display where users should not navigate out of the publication without permission. The kiosk unit does allow for keyboard and mouse use via admin access. Is there a potential option for full screen mode to respond directly to the screen aspect ratio and not include a border around the content? Thank you!

**Comment by danielweck** [comment]
Hello again, Thorium currently doesn't implement a suitable "kiosk" mode. I am afraid we have other pressing development goals on our roadmap at the moment. We are a very small technical team so we must prioritise.

However the use-case you are describing makes a lot of sense for an application like Thorium. It would be great if we could figure out a way to get this shipped in a not-too-distant release! Could you please give us a more detailed brief of what features (if any) would need to be disabled / removed from the regular user experience in the reader window? (for example, the ability to change display styles, add/remove bookmarks, save/reset settings, etc.) Please explore the menus and let know what you have in mind.

Thank you!

**Comment by danielweck** [comment]
> we're hoping to refine the scaling issues.

Note that Thorium 1.8 ships with a bugfix for the fuzzy text in scaled fixed-layout publications.
